src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationService.kt

```diff
@@ -34,16 +36,21 @@ class TierCalculationService(
     }
 
   private suspend fun calculateTier(crn: String): TierCalculationEntity {
+    val tierToDeliusResponse = tierToDeliusApiService.getTierToDelius(crn)
+
     val offenderAssessment = assessmentApiService.getRecentAssessment(crn)
-    val (rsr, ogrs) = communityApiService.getDeliusAssessments(crn)
+
+    val rsr = tierToDeliusResponse.rsrscore ?: BigDecimal.ZERO
```

Comment:
would be good to have this logic in the `tierToDeliusApiService`, some kind of domain pojo returned